components/LoanForm/LoanFormBetterTerms/LoanFormBetterTerms.tsx

```diff
@@ -206,7 +206,17 @@ export function LoanFormBetterTerms({
           }}
           balance={balance}
           accrued={formattedInterestAccrued}
-          totalPayback={formattedTotalPayback}
+          totalPayback={[
+            truncate(
```

Comment:
what does this truncate it to?

Reply:
not quite sure i am following the pattern of other formatted amounts from useLoanDetails

Reply:
looks like it adds ... if string length is longer than certain char count

Reply:
`useLoanDetails` defines a function coincidentally called `truncate`:
```ts
function truncate(num: string, maxDigits?: number): string;
function truncate(num: number, maxDigits?: number): string;
function truncate(num: string | number, maxDigits: number = 4): string {
  const number = typeof num === 'string' ? parseFloat(num) : num;
  const delta = number - Math.floor(number);
  // TODO: handle very small numbers that may render as 0.0000
  return number.toFixed(delta > 0 ? maxDigits : 0);
}
```

Reply:
at any rate, I think Wilson wanted these to render with as many decimals as the asset supports?

Reply:
oops thanks for that find, that was careless of me, ya lets remove truncate altogether